### ISSUE TITLE
Blauberg bathroom fan support?

### ISSUE THREAD

**Issue by jrb80** [gody01/ecovent_v2#11]
Question rather than issue, I am looking to buy this [Blauberg bathroom fan](https://www.blauberg.co.uk/en/amfile/file/download/file/814/product/4886/) (SMART-WIFI-CON) but as it's not a MVHR unit I was wondering whether it is possibly supported by this interface? As far as I can tell it uses the same [Blauberg Home](https://play.google.com/store/apps/details?id=com.blauberg.universal&hl=en_GB&gl=US) mobile application and therefore I gather it uses the same V2 protocol? Extract from compatibility list below. Thoughts?

![blauberg](https://user-images.githubusercontent.com/1820167/226122130-d268d81c-12ad-4f10-b235-a625d0a5c071.jpg)

**Comment by Styx85** [comment]
I don't think it would is the exact same protocol but you can try this: https://github.com/ynsgnr/blauberg-assistant

**Comment by Komzpa** [comment]
Hi @jrb80, this should now be testable in release `1.2.6`.

The merged work in #44 adds a dedicated `0x0600` Smart Wi-Fi / iFan extract-fan profile based on the Blauberg Smart Wi-Fi B168 protocol guide. This is not treated as a normal VENTO/TwinFresh unit anymore, because the parameter map is different.

Could you please try `1.2.6` with your Smart Wi-Fi extract/bathroom fan and report whether setup and basic control work? If it works, I think this issue can be closed. If something still fails, fresh logs plus the detected unit type would help.
